cmd/contour-operator.go

```diff
@@ -61,6 +64,15 @@ func main() {
 
 	ctrl.SetLogger(zap.New(zap.UseDevMode(true)))
 
+	images := []string{contourImage, envoyImage}
+	for _, image := range images {
+		_, err := reference.Parse(image)
+		if err != nil {
+			setupLog.Error(err, "invalid image reference", "value", image)
```

Comment:
While we are touching this, let's make `setupLog` local, since it doesn't need to be a global var.